server/plugin.go

```diff
@@ -73,74 +73,74 @@ func (p *Plugin) OnActivate() error {
 	if err != nil {
 		return errors.Wrap(err, "failed to reconfigure configurator on startup")
 	}
-	p.mm.Log.Debug("initialized config service")
-
-	accessKey := os.Getenv("APPS_INVOKE_AWS_ACCESS_KEY")
-	if accessKey == "" {
-		p.mm.Log.Warn("APPS_INVOKE_AWS_ACCESS_KEY is not set. AWS apps won't work.")
+	conf := p.conf.GetConfig()
+	mode := "Self-managed"
+	if conf.MattermostCloudMode {
+		mode = "Mattermost Cloud"
 	}
-	secretKey := os.Getenv("APPS_INVOKE_AWS_SECRET_KEY")
-	if secretKey == "" {
-		p.mm.Log.Warn("APPS_INVOKE_AWS_SECRET_KEY is not set. AWS apps won't work.")
+	if conf.DeveloperMode {
```

Comment:
what does it actually mean (practically) to be in both `MattermostCloudMode` and `DeveloperMode`?

Reply:
They are not mutually exclusive.

If an server is in `MattermostCloudMode` and in `DeveloperMode` it means that:
- There will be debug command like `/apps debug-list-bindings`
- `manifests.json` will be used to fetch manifest from S3
- There will only be a `/apps install marketplace` command